var/spack/repos/builtin/packages/freesasa/package.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Freesasa(AutotoolsPackage):
+    """C-library for calculating Solvent Accessible Surface Areas"""
+
+    homepage = "http://freesasa.github.io"
+    url = "https://github.com/mittinatten/freesasa/releases/download/2.1.2/freesasa-2.1.2.zip"
+    git = "https://github.com/mittinatten/freesasa.git"
+    maintainers("RMeli")
+
+    version("2.1.2", tag="2.1.2", branch="master", submodules=True)
```

Comment:
Can we use the add a sha256sum here for the version (instead of a floating tag)?